libs/openFrameworks/types/ofBaseTypes.h

```diff
@@ -848,15 +1681,18 @@ class ofBaseRenderer{
 	/// \param depth The depth of the box.
 	virtual void drawBox( float x, float y, float z, float width, float height, float depth) const;
 
-	/// \brief Draws a cube with the specified size, starting from the specified coordinates.
+	/// \brief Draws a cube using x, y, z, and size.
+	/// coordinates.
 	///
 	/// A cube is a rectangular solid bounded by six square faces of equal size.
-	/// It is also known as a regular hexahedron, a square parallelepiped, an equilateral cuboid
-	/// and a right rhombohedron. It is a regular square prism in three orientations.
+	/// It is also known as a regular hexahedron, a square parallelepiped, an
+	/// equilateral cuboid and a right rhombohedron. It is a regular square
+	/// prism in three orientations.
 	///
-	/// It is drawn starting from a 3D reference coordinate, with the specified size.
-	/// The cube is drawn with the current color, e.g. set with ofSetColor().
-	/// The cube is drawn filled by default; change this with ofFill();
+	/// It is drawn starting from a 3D reference coordinate, with the specified
```

Comment:
This description was already in the inline documentation. It may belong more in markdown rather than inline.